packages/admin/components/atomic/navs/MobileSubNav/MobileSubNav.tsx

```diff
@@ -38,7 +38,9 @@ const MobileSubNav = ({ tabRoutes, sidebarLinks }: Props) => {
         ref={mobileNavRef}
       >
         <MobileMenuList>
-          {tabRoutes?.map(({ label = "", ...namedLinkProps }, i) => (
+          {/* Do not pass actions down to the DOM */}
+          {/* eslint-disable-next-line @typescript-eslint/no-unused-vars */}
+          {tabRoutes?.map(({ label = "", actions, ...namedLinkProps }, i) => (
```

Comment:
What do you think about using an ignore pattern rather than disable comments? Something like...
```
// in rules in eslint.config.js

"@typescript-eslint/no-unused-vars": [
      "error",
      {
        varsIgnorePattern: "^_",
        argsIgnorePattern: "^_",
        destructuredArrayIgnorePattern: "^_",
        ignoreRestSiblings: true,
        caughtErrors: "none",
      },
    ],
```